src/content/web/wasm.md

```diff
@@ -0,0 +1,92 @@
+---
+title: WebAssembly (Wasm) compilation
+description: Learn how to compile your Dart web app to WebAssembly.
+---
+
+The Dart team is excited to add
+[WebAssembly](https://webassembly.org/) as a compilation target when building
+Dart and [Flutter][] applications for the web.
+
+Development of WebAssembly support remains ongoing,
+which will potentially result in frequent changes. 
+Revisit this page for the latest updates.
+
+:::note
+**Support for Wasm is now in stable!**
+
+WebAssembly support for Dart web is available on the Dart 
+*stable* [channel](/get-dart#release-channels).
+:::
+
+## WebAssembly support
+
+The current version of Dart compilation to WebAssembly has a number of
+restrictions:
+
+1. It targets WebAssembly with Garbage Collection ([WasmGC][]), so...
```

Comment:
", so not all browsers are currently supported. The current list of browsers can available in the [Flutter documentation](https://docs.flutter.dev/platform-integration/web/wasm#chrome-119-or-later)"